### DISCUSSION TITLE
ESLint shows "utilities are not ordered" when the class is wrapped into multiple lines

### DISCUSSION THREAD

**Discussion by ouuan** [unocss/unocss#2224] [Q&A]
Sometimes the class is very long so I need to wrap it into multiple lines to reduce the line length, but ESLint shows "utilities are not ordered" when the class string is wrapped.

![screenshot not ordered](https://user-images.githubusercontent.com/30581822/220242043-6e8d7d01-8109-460a-9ec9-0364433c243c.png)

![screenshot max-len](https://user-images.githubusercontent.com/30581822/220242298-4af4c018-7866-4ec8-88e7-85dceb5cea8f.png)

A workaround is to use an array instead of a string as the class. But is there any better solution, or can this be fixed in `@unocss/eslint-config`?

**Comment by SnowingFox** [comment]
You can try to fix this problem or use this [transformer](https://github.com/unocss/unocss/tree/main/packages/transformer-directives)
